content/en/docs/tutorials/hello-minikube.md

```diff
@@ -142,7 +142,7 @@ recommended way to manage the creation and scaling of Pods.
     kubectl config view
     ```
 
-1. View application logs for a container in a pod.
+1. View application logs for a container in a pod. Make sure to replace `hello-node-5f76cf6ccf-br9b5` with the name of the pod from `kubectl get pods` command output.
```

Comment:
```suggestion
1. View application logs for a container in a pod. Make sure to replace
   `hello-node-5f76cf6ccf-br9b5` with the name of the pod from `kubectl get pods` command output.
```
We can follow the context style to wrap long lines.

Reply:
Would it be better to wrap it at the start of the sentence?

Reply:
All right, that will be 3 lines.

Reply:
How do I edit your suggested version to suggest my change? I can only find a way to edit my original PR.

Reply:
Edit it locally and run the following commands to submit again:

```
git add .
git commit --amend --no-edit
git push -f
```

Reply:
I just tried it in the browser, I don't think it ended up in the right place but I've made the change.